### ISSUE TITLE
Icon for CSS files is not displayed in Sidebar

### ISSUE THREAD

**Issue by eduludi** [equinusocio/material-theme#1000]
### Summary

![screen shot](https://cloud.githubusercontent.com/assets/76132/19468031/a6ed40e4-9514-11e6-9f53-ad3e0cb90d71.png)

CSS icon is not being displayed in the sidebar. All other (tested)  icons are being displayed correctly.

Issue persists after reinstalling the package.

### Environment

- Sublime Text:   3126
- Platform:  macOS Sierra 10.12 (16A323)
- Arch:   x64
- Theme:   3.2.2
- Install via PC:   True
- Dependencies:
    * mdpopups:   1.10.0
    * markdown:   2.6.6
    * pygments:   2.1a0
    * jinja2:   2.8

### Material Theme Settings

```json
{ 
	"theme": "Material-Theme-Darker.sublime-theme",
	"color_scheme": "Packages/Material Theme/schemes/Material-Theme-Darker.tmTheme",
	"material_theme_accent_orange": true,
	"material_theme_contrast_mode": true,
	"material_theme_panel_separator": true,
	"material_theme_small_statusbar": true,
	"material_theme_tabs_autowidth": true
}
```

**Comment by equinusocio** [comment]
Please wait 4.0.0 release. From 4.0.0 Material Theme will use [zz File Icons](https://github.com/oivva/zz-file-icons) package (that is under review by Package Control). When this package will be released on PC i will release the 4.0.0 that bring a lot of fixes (also new file icons) and new features, and the new Palenight color theme. 

So, from the next release (4.0.0) you will have to install [zz File Icons](https://github.com/oivva/zz-file-icons) to display file icons.

**Comment by equinusocio** [comment]
This should be fixed from the latest 4.0.0 release.
